src/blog/2024-11-02/index.md

```diff
@@ -0,0 +1,107 @@
+* Tachi and Milso can no longer shoot underwater
+* Corrected issue with new NEOUI and legacy settings being out of sync
+* Fixed issue with respawning once round is live vs in freeze time
+* Added M41L (inaccessible in weapon selection menu; use `give weapon_m41l` with `sv_cheats 1` to use)
+* Fixed score cap in TDM/DM
+* Fixed damage/penetration values for SRM, SRMS, ZR68C to match original NEOTOKYO
+* Fixed muzzle flash from showing when scoped in
+* Changes to ready up system 
+    * Changed command prefix from `!` to `.`
+    * Changed command names to match original NEOTOKYO ready up commands. 
+        * `.playersnotready` is now `.readylist`
+        * `.overridelimit` is now `.start`
+    * Fixed spectators showing in `.readylist`
+    * `.help` now outputs to chat
+    * Added `.pause` & `.unpause` commands
+        * Menu is displayed to choose between a short (30 seconds) or long (3 minute) pause
+        * Pauses can happen instant if during a pre-round freeze, otherwise will just goes into the next round
+        * Cannot pause during idle/warmup as that is not a live match
+        * HUD updated and center text flashing to indicate pause state. Pause state have it red in the HUD round state
+        * The team that paused can start unpause request with `.unpause`. Then the non-pausing team is required to respond with `.unpause`. The pausing team can cancel the unpause request by `.unpause`
+    * New ConVars:
+        * `neo_sv_pausematch_enabled` - 0 by default, enable match pausing feature.
+        * `neo_sv_pausematch_unpauseimmediate` - 0 by default, locked by cheat flag, unpause command is immediate.
+* Fixed knife from swinging with `+attack2`
+* Removed startup intro video
+* Fixed issue with spectator x-ray being visible on respawn
+* Corrected x-ray draw
+* Added placeholder missing weapon animations
+* Added `cl_neo_bullet_trace` (cheat protected) - Show bullet trace
+* Added `cl_neo_bullet_trace_max_pen` (cheat protected) - How much pen does a bullet need to have to show up as a solid red line. Configure to the current weapon used, or leave on default to see differences in pen between weapons
+* Rangefinder restored from original NEOTOKYO
+    * Added `neo_cl_hud_rangefinder_enabled` - 1 as default, toggle enable/disable
+    * Added `neo_cl_hud_rangefinder_pos_frac_[x/y]` - 0.55 as default, position in fraction in screen width/height
+* Corrected Jinrai recon sprinting animations
+* Compass fade commands have been restored
+* Fixed SRS weapon reload timing
+* Added `neo_sv_gamemode_enforcement` - Default = 0 (map - neo_game_config), determine gamemode by... 0 = map (neo_game_config), 1 = neo_sv_gamemode_single, 2 = Random within neo_sv_gamemode_random_allow, 3 = vote (neo_vote_game_mode in idle/warmup)
+* Added `neo_sv_gamemode_single` - The single game mode to enforce
+* Added `neo_sv_gamemode_random_allow` - The bitwise of game modes that is allowed to be used for the randomizer
```

Comment:
Put in: `* Gamemode enforcement by default is now changed to by map. Singular/random/by vote also implemented.` (something like this, not exact grammatically)

Then sub-section the convars bullet points